### DISCUSSION TITLE
Repository Ideas

### DISCUSSION THREAD

**Discussion by CyberGlitch01** [Innovative-CST/Android-Code-Editor#14] [General]
Can we move the main branch to the dev branch for development and production release?
Main branch for last release and dev branch for latest development.

**Comment by CyberGlitch01** [comment]
Message to all Contributors and members.
@moderGamer @PHANTOMLOCAL @DevKumar00001 @TechnicalStudioDeveloper2

**Reply by DevVigilante** [reply]
Nice idea but currently not possible
From next release this will be better

**Comment by DevVigilante** [comment]
Idea: Updating new project structure according to classes extending and types

**Reply by CyberGlitch01** [reply]
Nice idea

**Comment by CyberGlitch01** [comment]
@DevKumar00001 should I update project structure?

**Reply by DevVigilante** [reply]
Okay

**Comment by CyberGlitch01** [comment]
It is ok to put Sketchware request network classes in utils or keep it in a Sketchware package as now

**Reply by DevVigilante** [reply]
Move as utils

**Comment by DevVigilante** [comment]
> Can we move the main branch to the dev branch for development and production release? Main branch for last release and dev branch for latest development.

From now we will push commits to the dev branch and only push it to main branch if there is no error in new features
 @PHANTOMLOCAL , @CyberGlitch01

**Comment by DevVigilante** [comment]
@moderGamer Can you come at discord?
I can't come online at telegram my account in not activing somehow I can't login.
